charts/temporal-operator/values.yaml

```diff
@@ -0,0 +1,27 @@
+controllerManager:
```

Comment:
IMO we can remove the `controllerManager` prefix. Looks `manager` is sufficient.

Reply:
Renamed `controllerManager.manager` -> `manager`